src/components/modal/README.md

```diff
@@ -471,6 +472,35 @@ To disable stacking for a specific modal, just set the prop `no-stacking` on the
 - The opaque backdrop will appear progressively darker for each modal that is opened. This is expected behaviour as each backdrop is opened over top the other backdrops.
 - For multiple modals to stack properly, they **must** be defined in the document in the order they will be opened, otherwise a newly opened modal may appear hidden or obscured by a currently open modal.
 
+
+## Listen for a modal changes via $root events
```

Comment:
This should be noted that it is the `v-b-toggle` directive that emits this event, not he modal itself.

If a modal is opened programatically via $refs methods (i.e. `this.$refs.mymodal.show()`), or using the v-model, etc, this event **will not** be emitted.